packages/ui-pages/babel.config.js

```diff
@@ -23,11 +23,6 @@
  */
 
 module.exports = {
-  shouldPrintComment(value) {
-    const isTsRelatedComment = /@ts-expect-error|FIXME/.test(value)
-
-    return !isTsRelatedComment
-  },
```

Comment:
Should we remove this, since the locator file still has the ts-expect-errors?

Reply:
The Pages component doesn't have locators, so there are no more ts-expect-errors in the package.